containers/cloud-gateway-service/Dockerfile

```diff
@@ -0,0 +1,40 @@
+#######################################################################
+# This program and the accompanying materials are made available
+# under the terms of the Eclipse Public License v2.0 which
+# accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright Contributors to the Zowe Project.
+#######################################################################
+
+# base image tag
+ARG ZOWE_BASE_IMAGE=latest-ubuntu
+
+FROM zowe-docker-release.jfrog.io/ompzowe/base-jdk:${ZOWE_BASE_IMAGE} AS builder
+
+##################################
+# labels
+LABEL name="Cloud Gateway" \
+      maintainer="carson.cook@ibm.com" \
```

Comment:
Is this contact person correct? Is it published somewhere?

Reply:
We should probably change this to someone else from the Squad.

Reply:
I think it will be a part of the image. As I don't know whom to put here as a contact, I would rather stay with the existing one and potentially change it in the future in all of these files